src/pyodide-py/pyodide/_base.py

```diff
@@ -49,9 +49,12 @@ class CodeRunner:
 
     Parameters
     ----------
-    ns
-        `locals()` or `globals()` context where to execute code.
-        This namespace is updated by the subsequent calls to `run()`.
+    globals
+        The global scope in which to execute code. This is used as the `exec` `globals` parameter.
+        See [the exec documentation](https://docs.python.org/3/library/functions.html#exec) for more info.
+    locals
+        The local scope in which to execute code. This is used as the `eval` `globals` parameter.
```

Comment:
`eval` `locals` parameter. ?

Also indicate that when not provided `globals` parameter is used.